scripts/us_usda/quickstats/manifest.json

```diff
@@ -0,0 +1,16 @@
+{
+    "import_specifications": [
+        {
+            "import_name": "UsdaAgSurvey",
+            "curator_emails": ["keyurs@google.com"],
+            "provenance_url": "https://quickstats.nass.usda.gov/api/",
+            "provenance_description": "The National Agricultural Statistics Service (NASS) offers Quick Stats, an on-line database containing official published aggregate estimates related to U.S. agricultural production.",
+            "scripts": ["process.py"],
+            "import_inputs": [{
+                "template_mcf": "ag.tmcf",
+                "cleaned_csv": "output/*.csv"
```

Comment:
I'm not sure if the import automation supports wildcards. 
Since the  script produces a single csv for all years data, can the year be dropped from the filename?

Reply:
It produces 1 file per year right now.